sdk/tables/azure-data-tables/cgmanifest.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "https://json.schemastore.org/component-detection-manifest.json",
+  "Registrations": [
+    {
+      "Component": {
+        "Type": "git",
+        "git": {
+          "RepositoryUrl": "https://github.com/cpplint/cpplint",
+          "CommitHash": "8456631ba3a12842da08238362a90d33fcf47062"
+        }
+      },
+      "DevelopmentDependency": true
+    },
+    {
+      "Component": {
+        "Type": "other",
+        "Other": {
+          "Name": "clang-format",
+          "Version": "9.0.0-2",
+          "DownloadUrl": "https://ubuntu.pkgs.org/18.04/ubuntu-updates-universe-amd64/clang-format-9_9-2~ubuntu18.04.2_amd64.deb.html"
+        }
+      },
+      "DevelopmentDependency": true
+    },
+    {
+      "Component": {
+        "Type": "other",
+        "Other": {
+          "Name": "doxygen",
+          "Version": "1.8.20",
+          "DownloadUrl": "http://doxygen.nl/files/doxygen-1.8.20-setup.exe"
+        }
+      },
+      "DevelopmentDependency": true
+    },
+    {
+      "Component": {
+        "Type": "git",
+        "git": {
+          "RepositoryUrl": "https://gitlab.gnome.org/GNOME/libxml2",
+          "CommitHash": "da703eaaea152f6dd9e871cbe7f0ae46322c583d"
+        }
+      },
+      "DevelopmentDependency": false
```

Comment:
~Do we use libxml2 in data-tables or is it JSON based?~

**Edit:** Nevermind, I see we have to parse both XML and JSON here.